# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nss_ldap/nss_ldap-238-conffile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
## nss_ldap-238-conffile.patch by Peter Marschall <peter@adpm.de>
##
## revert 'strange' changes to ldap.conf

--- ./ldap.conf
+++ ./ldap.conf	2005-03-24 20:09:13.132764983 +0100
@@ -162,32 +162,32 @@
 
 # RFC2307bis naming contexts
 # Syntax:
-# nss_base_XXX	base?scope?filter
+# nss_base_XXX		base?scope?filter
 # where scope is {base,one,sub}
 # and filter is a filter to be &'d with the
 # default filter.
 # You can omit the suffix eg:
-# nss_base_passwd ou=People,
+# nss_base_passwd	ou=People,
 # to append the default base DN but this
 # may incur a small performance impact.
-#nss_base_passwdou=People,dc=padl,dc=com?one
-#nss_base_shadowou=People,dc=padl,dc=com?one
-#nss_base_group	ou=Group,dc=padl,dc=com?one
-#nss_base_hosts	ou=Hosts,dc=padl,dc=com?one
-#nss_base_servicesou=Services,dc=padl,dc=com?one
-#nss_base_networksou=Networks,dc=padl,dc=com?one
-#nss_base_protocolsou=Protocols,dc=padl,dc=com?one
-#nss_base_rpc	ou=Rpc,dc=padl,dc=com?one
-#nss_base_ethersou=Ethers,dc=padl,dc=com?one
-#nss_base_netmasksou=Networks,dc=padl,dc=com?ne
-#nss_base_bootparamsou=Ethers,dc=padl,dc=com?one
-#nss_base_aliasesou=Aliases,dc=padl,dc=com?one
-#nss_base_netgroupou=Netgroup,dc=padl,dc=com?one
+#nss_base_passwd	ou=People,dc=padl,dc=com?one
+#nss_base_shadow	ou=People,dc=padl,dc=com?one
+#nss_base_group		ou=Group,dc=padl,dc=com?one
+#nss_base_hosts		ou=Hosts,dc=padl,dc=com?one
+#nss_base_services	ou=Services,dc=padl,dc=com?one
+#nss_base_networks	ou=Networks,dc=padl,dc=com?one
+#nss_base_protocols	ou=Protocols,dc=padl,dc=com?one
+#nss_base_rpc		ou=Rpc,dc=padl,dc=com?one
+#nss_base_ethers	ou=Ethers,dc=padl,dc=com?one
+#nss_base_netmasks	ou=Networks,dc=padl,dc=com?ne
+#nss_base_bootparams	ou=Ethers,dc=padl,dc=com?one
+#nss_base_aliases	ou=Aliases,dc=padl,dc=com?one
+#nss_base_netgroup	ou=Netgroup,dc=padl,dc=com?one
 
 # attribute/objectclass mapping
 # Syntax:
-#nss_map_attributerfc2307attribute	mapped_attribute
-#nss_map_objectclassrfc2307objectclass	mapped_objectclass
+#nss_map_attribute	rfc2307attribute	mapped_attribute
+#nss_map_objectclass	rfc2307objectclass	mapped_objectclass
 
 # configure --enable-nds is no longer supported.
 # NDS mappings
